js/templates/modals/settings/advanced/server.html

```diff
@@ -0,0 +1,45 @@
+<form class="box padKids padStack clrP clrBr js-server">
+  <div class="flexRow">
+    <h3><%= ob.polyT('settings.server') %></h3>
+  </div>
+  <div class="flexRow">
+    <div class="col3">
+      <label for="currentFirewallType">
+        <%= ob.polyT('settings.currentFirewallType') %>
+      </label>
+    </div>
+    <div class="col9">
+      <input class="clrBr" readonly
+        type="text"
+        name="currentFirewallType" 
+        id="currentFirewallType" 
+        value="<%= ob.currentFirewallType %>">
+    </div>
+  </div>
+  <div class="flexRow">
+    <div class="col3">
+      <label>
+        <%= ob.polyT('settings.connections') %>
+      </label>
+    </div>
+    <div class="col9">
+      <div class="btnStrip flex clrP clrBr">
+        <a class="btn clrP clrBr js-showConnectionManagement flexNoShrink"><%= ob.polyT('settings.btnShowConnectionManagement') %></a>
+        <a class="btn clrP clrBr js-showConnectedPeers flexNoShrink"><%= ob.polyT('settings.btnShowConnectedPeers') %></a>
+      </div>
+    </div>
+  </div>
+  <div class="flexRow">
+    <div class="col3">
+      <label>
+        <%= ob.polyT('settings.onOff') %>
+      </label>
+    </div>
+    <div class="col9">
+      <div class="btnStrip flex clrP clrBr">
```

Comment:
Yes, you can have the client open without being connected to a server, that's how you connect to remote servers.